user/frr/files/tmpfiles.conf

```diff
@@ -0,0 +1,6 @@
+d /run/frr              0750 _frr _frr -
+Z /usr/share/frr/conf/* 0640 _frr _frr
```

Comment:
tmpfiles should not alter modes of stuff in `/usr`

nor it's particularly desirable for it to, all `/usr` can be world readable